sample_tracking/app/i18n/locales/pt/translations.json

```diff
@@ -85,6 +85,14 @@
   "latLonRequired": "lat e lon são obrigatórios,",
   "shouldBeWithinTheRange": "deve estar dentro do intervalo",
   "and": "e",
+  "errorCreatingAccount": "Houve um erro ao criar a sua conta.",
+  "passwordsDontMatch": "As senhas que você digitou não coincidem.",
+  "incorrectLogin": "O nome de usuário ou senha inseridos estão incorretos.",
+  "invalidEmail": "Por favor digite um email válido",
+  "emailAlreadyInUse": "Já existe uma conta com este e-mail.",
+  "editSample": "Editar amostra",
+  "municipality": "Município",
+  "weakPassword": "A senha deve ter pelo menos 6 caracteres.",
```

Comment:
you will want to have a native speaker verify these translations as its very likely some connotations are off.

Reply:
Good point- I'll ask some of the native speakers on the team to take a look at the translations.